theano/tensor/tests/test_opt.py

```diff
@@ -5675,18 +5675,19 @@ def test_local_useless_split():
 
 def test_local_flatten_lift():
     for i in xrange(1, 4):
-        op = tensor.Flatten(i)
         x = tensor.tensor4()
-        out = op(T.exp(x))
+        out = tensor.flatten(T.exp(x), i)
         assert out.ndim == i
         mode = compile.mode.get_default_mode()
         mode = mode.including('local_flatten_lift')
```

Comment:
I would keep it if we reload pickled graph. But add a comment to it why we keep it and that it will be removed in the future. We should remove it at the same time as the Flatten class, so that old pickled files will just crash instead of running non optimized (or we should make an optimization to convert it to Reshape and remove that optimization?, I would go for the faster to implement for now)